src/Plugin.ts

```diff
@@ -99,8 +100,19 @@ export class Plugin {
     );
 
     await networkObservable.subscribe(
-      async (request: NetworkRequest): Promise<number> =>
-        this.entries.push(await new EntryBuilder(request).build())
+      async (request: NetworkRequest): Promise<number> => {
+        const requestPath = request.parsedURL.path;
```

Comment:
Use  destructuring assignment syntax instead to avoid such long reference to variables
```suggestion
        const { path } = request.parsedURL;
```